techniques/general/G209.html

```diff
@@ -2,150 +2,126 @@
+            <li><a href="#resources">関連リソース</a></li>
+            <li><a href="#related">関連する達成方法</a></li>
+            <li><a href="#tests">検証</a></li>
          </ul>
       </nav>
-      <h1>Provide sufficient contrast at the boundaries between adjoining colors</h1>
+      <h1>隣り合う色の境界に十分なコントラストを設定する</h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://w3c.github.io/wcag/understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
-         </p>
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://w3c.github.io/wcag/understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。</p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>For graphics which use multiple colours adjacent to each other.</p>
-            <p>This technique relates to <span><a href="https://w3c.github.io/wcag/understanding/non-text-contrast">Success Criterion 1.4.11: Non-text Contrast</a> (Sufficient)</span>.
-            </p>
+            <h2>適用 (対象)</h2>
+            <p>互いに隣接し合う複数の色を使用したグラフィックス (図)。</p>
+            <p>これは、<span><a href="https://w3c.github.io/wcag/understanding/non-text-contrast">達成基準 1.4.11 非テキストのコントラスト</a> (十分) </span> に関連する達成方法である。</p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to ensure people with moderately low vision can
-               distinguish boundaries between adjoining segments of color.
-            </p>
-            <p>The success criteria for non-text contrast uses the term "graphical object" to cover
-               small simple graphics, and parts of larger complex graphics. This technique focuses
-               on solid color segments where the boundaries between colors convey meaning.
-            </p>
-            <p>When selecting colors for graphics with multiple colors, consider adjoining colors
-               and test that the contrast ratio is at least 3:1. If adjoining colors have less than
-               3:1 color contrast ratio difference add a border with at least a 3:1 color contrast
-               with each color.
+            <h2>解説</h2>
+            <p>このテクニックの目的は、中程度のロービジョンの人が、境界が隣り合う色の区切り (セグメント) を確実に識別できることである。
```

Comment:
```suggestion
            <p>この達成方法の目的は、中程度のロービジョンの人が、隣り合う色の区分間にある境界を確実に識別できることである。
```
G209を見る限りでは、segmentsは区切りというよりかはパーツのイメージなのではないでしょうか、ということで区分としてみました。

Reply:
うまい言い回しに迷いました。ありがとうございます。